map/autoware_lanelet2_map_validator/test/src/map_validation_tester.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MAP_VALIDATION_TESTER_HPP_
+#define MAP_VALIDATION_TESTER_HPP_
+
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <autoware_lanelet2_extension/projection/mgrs_projector.hpp>
+
+#include <gtest/gtest.h>
+#include <lanelet2_io/Io.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+class MapValidationTester : public ::testing::Test
+{
+protected:
```

Comment:
```suggestion
protected:
virtual void SetUp() {
  map_ = lanelet::load(package_share_directory + "/data/map/" + file_name, *projector, &loading_errors_);
```
}

Reply:
Isn't this difficult to pass the target map file to validate?
One test code will try to validate various maps.

Reply:
Please forget my comment. Possibly we can use `TestWithParam` in the future when the number of test suites grow.
https://github.com/google/googletest/blob/main/docs/advanced.md#how-to-write-value-parameterized-tests